wordpress-mobile/WordPress-iOS
# Fix stats not updating when switching sites

Fixes #14759

applies the same fix of #14801 to release 15.6

To test:

pre-requisite: have an account with more than one site and some activity that populates the stats (in at least one of them)

- Select a site and go to stats
- Observe charts and values in any of the tabs (particularly the periods)
- Go back to My site, Switch site then select a different one
- Go to Stats and make sure that they are consistent with the site you chose, and are not from the previous site

PR submission checklist:
